### PR TITLE
bump shopify version

### PR DESCRIPTION
**Pull request by csaunders**
All the deps are gone and we need to release a new gem.

Please review @camilo

### CODE REVIEW

File: lib/shopify_api/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ShopifyAPI
-  VERSION = "3.2.7"
+  VERSION = "4.0.0"
```
Comment (camilo) [inline lib/shopify_api/version.rb]: needs tests :trollface:

**Comment by camilo** [comment]
:+1:

**Comment by dylanahsmith** [comment]
@csaunders why was the major version bump done?  There is no entry for it in the changelog.

cc @clayton-shopify

**Comment by csaunders** [comment]
We removed the CLI features from the gem.

**Pull request merged**